packages/web-react/src/components/Button/Button.tsx

```diff
@@ -3,8 +3,7 @@ import classNames from 'classnames';
 
 type Color = 'primary' | 'secondary' | 'tertiary';
 
-const getButtonColorClassname = (color: Color): string =>
-  `lmc-Button--${color}`;
+const getButtonColorClassname = (color: Color): string => `Button--${color}`;
```

Comment:
There will be no prefix in the components? There could be a general Context for this, where it will be set up and all the other components will take over. https://reactjs.org/docs/context.html

Reply:
in twig we will be able to set it at the bundle level

Reply:
I removed it for now so `web-react` works with current `web`. I'm going to extend `web-react` components with context provider/consumer exactly as you suggest, just in a standalone PR (or issue, should it turn out I don't have the skill to do it right).